pkgs/os-specific/linux/udisks/2-default.nix

```diff
@@ -22,6 +22,10 @@ stdenv.mkDerivation rec {
       substituteInPlace src/main.c --replace \
         "@path@" \
         "${utillinux}/bin:${mdadm}/sbin:/var/run/current-system/sw/bin:/var/run/current-system/sw/bin"
+      substituteInPlace data/80-udisks2.rules \
+        --replace "/bin/sh" "${stdenv.shell}" \
+        --replace "/sbin/mdadm" "${mdadm}/sbin/mdadm" \
+        --replace " sed " " ${gnused}/bin/sed "
```

Comment:
From looking at the patch it is just a by-default PATH if no is set. I wonder if we perhaps want to set it always instead of as a fallback...